composer.json

```diff
@@ -10,7 +10,7 @@
         }
     ],
     "require": {
-        "php": ">=5.5.9",
+        "php": ">=7.2",
```

Comment:
@chaoukibenfradj Does your change introduce anything that requires at least PHP >= 7.2? I don't think so (I may have missed something), so we should be fine leaving the minimum PHP requirement as it was before. As said this is just the minimum requirement, so we can still of course use the package with higher PHP versions.

Reply:
@PatrickBidx Yes we do need PHP>=7.* because the Error class that I added it, was introduced starting from PHP7.

Reply:
You are completely right. As `Error` was introduced in PHP 7.0, let's set the PHP requirement to 7.0.